docs/interpreter/hdfs.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: page
+title: "HDFS File System Interpreter"
+description: ""
+group: manual
+---
+{% include JB/setup %}
+
+## HDFS File System Interpreter for Apache Zeppelin
+
+[Hadoop File System](http://hadoop.apache.org/) is a distributed, fault tolerant file system part of the hadoop project and is often used as storage for distributed processing engines like [Hadoop MapReduce](http://hadoop.apache.org/) and [Apache Spark](http://spark.apache.org/) or underlying file systems like [Alluxio](http://www.alluxio.org/).
+
+## Configuration
+<table class="table-configuration">
+  <tr>
+    <th>Property</th>
+    <th>Default</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>hdfs.url</td>
+    <td>http://localhost:50070/webhdfs/v1/</td>
+    <td>The URL for WebHDFS</td>
+  </tr>
+  <tr>
+    <td>hdfs.user</td>
+    <td>hdfs</td>
+    <td>The WebHDFS user</td>
+  </tr>
+  <tr>
+    <td>hdfs.maxlength</td>
+    <td>1000</td>
+    <td>Maximum number of lines of results fetched</td>
+  </tr>
+</table>
+
+<br/>
+This interpreter connects to HDFS using the HTTP WebHDFS interface.
+It supports the basic shell file commands applied to HDFS, it currently only supports browsing
```

Comment:
Hi @runyontr : )
Could you add a new line after this sentence ?
Currently, the docs on webpage is shown like this below img.
![screen shot 2016-03-07 at 3 35 47 pm](https://cloud.githubusercontent.com/assets/10060731/13562294/578694ce-e47a-11e5-8927-c6cfb7c1be7f.png)